README.md

```diff
@@ -842,7 +843,16 @@ yarn remove prop-types
 }
 ```
 
-3. Finally add `.tsx` to the list of extensions in `config/webpacker.yml`
+3. Add `ts-loader` to `config/webpack/environment.js`
+
+```js
+environment.loaders.add('ts', {
+  test:  /.(ts|tsx)$/,
+  loader: 'ts-loader'
+})
+```
```

Comment:
I removed this step because we're adding all loaders by default now, but didn't notice the `tsx` extension part. Maybe we should just add it to the loader?
https://github.com/rails/webpacker/blob/3cd2e49b7943ddbea9da751a86b0ecdd80f2f61a/package/loaders/typescript.js#L1-L4

Reply:
@javan Oh yes, totally forgot. Yes, makes sense 👍